.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "awsm"]
 path = awsm
-url = https://github.com/gwsystems/aWsm
+url = https://github.com/lyuxiaosu/aWsm.git 
```

Comment:
I don't see any changes in your fork, so please be sure to revert this to gwsystems.

Reply:
> I don't see any changes in your fork, so please be sure to revert this to gwsystems.

Good catch, I didn't change anything for aWsm. The original thought was not want the update of the compiler to affect my branch:), finally, this won't be submitted.